lang/en/filter_opencast.php

```diff
@@ -35,3 +35,4 @@
 $string['setting_uselti_desc'] = 'When enabled, Opencast videos are delivered through LTI authentication using the <strong>default Opencast video player</strong>. This is typically used alongside Secure Static Files in Opencast for enhanced security.';
 $string['setting_uselti_nolti_desc'] = 'To enable LTI Authentication for Opencast, you must configure the required credentials (Consumer Key and Consumer Secret) for this instance. Please do so via this link: {$a}';
 $string['setting_uselti_ocinstance_name'] = 'Opencast API {$a} Instance';
+$string['no_engageurl_error'] = 'No URL available. Please check if the api endpoint is available and if the api user has the proper rights.';
```

Comment:
We need to use a more comprehensive (not-technical) info here, as it might be displayed to the end-users and they for example cannot check the api or the user rights!
What do you think?